src/components/views/settings/Notifications.js

```diff
@@ -65,82 +64,78 @@ function portLegacyActions(actions) {
     }
 }
 
-export default createReactClass({
-    displayName: 'Notifications',
-
-    phases: {
+export default class Notifications extends React.Component {
+    static phases = {
```

Comment:
Not for this pr but these phases are repeated in other files. Seems like there is a general pattern